### PR TITLE
allowing platform specification through flag

### PR DESCRIPTION
**Pull request by a-harhar**
- allows platform specification through new flag --platform
- if not platform defaults to local (empty string)
- we should use --platform linux/amd64 for csci350 (not sure which one for csci104)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,26 +142,28 @@ to wherever your homework is on your machine. For Windows, your volume command s
...
+       --image camerondurham/cs350-docker:v1 \
+       --volume PATH_TO_YOUR_WORKDIR:/xv6_docker \
+       --security-opt seccomp:unconfined \
+       --port 7776:22 \
+       --port 7777:7777 \
+       --port 25000:25000 \
+       --cap-add SYS_PTRACE \
+       --shell /bin/bash \
+       --privileged
+       --platform linux/amd64
```
Comment (camerondurham) [inline README.md:161]:
```suggestion
       --privileged \
       --platform linux/amd64
```

nit: needs an extra `\` for this command to work

**Review by camerondurham — COMMENTED** [review]
Looks great! Thanks for adding this so quickly. Just had a small nit on the csci350 command.

> not sure which one for csci104

It shouldn't be required for csci104 because the image is built for `linux/amd64` and `linux/arm64` already: https://github.com/csci104/docker/blob/main/.github/workflows/publish-image.yml#L34-L35

**Comment by a-harhar** [comment]
Thanks for looking it over! Committed your suggestion :) -- really enjoyed working on this though, it's a great tool you made!

**Review by camerondurham — APPROVED** [review]

**Comment by camerondurham** [comment]
Great, thanks! I'll need to update the README there once I minor version bump this to v0.3.7 so this gets built into the latest release.

**Pull request merged**